packages/net/tcp/auth.pony

```diff
@@ -0,0 +1,13 @@
+use "net"
+
+primitive TCPAuth
```

Comment:
I think this file should be named `tcp_auth.pony`, because the principal type declared in the file is `TCPAuth`.